README.md

```diff
@@ -275,6 +278,39 @@ let blue = ViewStyle(
 
 myButton.viewStyle = blue
 ```
+#### Layer properties
+If you set any of `[.borderColor, .borderWidth, .cornerRadius, .opacity, .shadow]` properties for particular states to `UITextView` or `UITextField`, the Styles will require to match properties under the hood. If you don't do so the styling becomes invalid because layer properties are not reset when state is changed. It is up to you to define the style. Below is the example which produces assertion error and second snippet is valid configuration.
+
+```swift
+let redBorder = ViewStyle(
+	.borderColor(.red),
+	.borderWidth(1.5)
+)
+
+let roundedCorners = ViewStyle(
+	.cornerRadius(10)
+)
+
+UITextField.appearance().setViewStyle(redBorder, for: .editing)
+UITextField.appearance().setViewStyle(roundedCorners, for: .inactive)
```

Comment:
pelase add comment here saying that this will raise

Reply:
is in the comment above the snippet but ok, will do so